lib/styles/modifiers/_colors.scss

```diff
@@ -1,35 +1,31 @@
-.-cadetGray {
-	color: $color-cadet-gray;
+.-gray400 {
+	color: var(--raw-gray-400);
 }
 
-.-catalinaBlue {
-	color: $color-catalina-blue;
+.-gray700 {
+	color: var(--raw-gray-700);
 }
 
-.-stormGray {
-	color: $color-storm-gray;
+.-gray300 {
+	color: var(--raw-gray-300);
 }
 
-.-mischkaGray {
-	color: $color-mischka-gray;
+.-white {
+	color: var(--raw-white);
 }
 
-.-totalWhite {
-	color: $color-total-white;
+.-green500 {
+	color: var(--raw-green-500);
 }
 
-.-jungleGreen {
-	color: $color-jungle-green;
+.-blue500 {
+	color: var(--raw-blue-500);
 }
 
-.-navyBlue {
-	color: $color-navy-blue;
+.-black {
+	color: var(--raw-black);
 }
 
-.-fireflyBlack {
-	color: $color-firefly-black;
-}
-
-.-delugeViolet {
-	color: $color-deluge-violet;
+.-violet500 {
+	color: var(--raw-violet-500);
 }
```

Comment:
I believe 1 class is missing. Was it intended?

Reply:
This class has no usage in DS and in Platform.

Reply:
@kvas-damian what do You think if we will remove all these classes? I will be better `clean up`. I checked usages in the Platform:
```
-gray400 - 5
-gray700 - 19
-gray300 - 4
-white - 0?
-green500 - 4
-blue500 - 0
-black  - 0
-violet500 - 0
```
All cases: 32.

I think we should replace like this:
```
<td class="-gray400">
```
to:
```
<td class="grayCell"> 
and
.gray { var(--raw-gray-400); }
```

Reply:
File removed.